presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -182,3 +182,8 @@ Floating Point Functions
 .. function:: nan() -> double
 
     Returns the constant representing not-a-number.
+
+.. function:: width_bucket(x, bound1, bound2, n) -> bigint
+
+    Returns the bin number of ``x`` in an equidepth histogram with the
```

Comment:
Whoops meant to write equi width there. I honk I was reading something about equi depth and had a brain slip